infrastructure/fivetran-stack.yml

```
@@ -23,6 +23,10 @@ Resources:
             - Effect: Allow
               Action: "secretsmanager:GetSecretValue"
               Resource: !ImportValue "opsGenieTokenSecretARN"
+            - Effect: Allow
+              Action: "kms:Decrypt"
+              Resource: !ImportValue "SlackApiTenantKmsKeyId"
```

```suggestion
              Fn::ImportValue: !Sub "SlackApiTenantKmsKeyId-${StackOrganisationId}"
```
!ImportValue and !Sub cannot be used together, have to use Fn::ImportValue